ivy/functional/frontends/torch/nn/functional/pooling_functions.py

```diff
@@ -103,27 +103,41 @@ def avg_pool2d(
 
 
 @to_ivy_arrays_and_back
-def max_pool1d(input, kernel_size, stride=None, padding=0):
+def max_pool1d(
+    input,
+    kernel_size,
+    stride=None,
+    padding=0,
+    ceil_mode=False,
+    dilation=1,
+    return_indices=False,
+):
+    if stride is None:
+        stride = kernel_size
     kernel_size = _broadcast_pooling_helper(kernel_size, "1d", name="kernel_size")
     stride = _broadcast_pooling_helper(stride, "1d", name="stride")
     padding = _broadcast_pooling_helper(padding, "1d", name="padding")
```

Comment:
Please check whether we need to use ` _broadcast_pooling_helper` here. I think the broadcasting would be done inside the ivy function later on.

Reply:
okay on it.